platform_api/cluster_config_factory.py

```diff
@@ -45,12 +54,15 @@ def _create_ingress_config(self, payload: dict[str, Any]) -> IngressConfig:
     def _create_presets(self, payload: dict[str, Any]) -> list[Preset]:
         result = []
         for preset in payload.get("resource_presets", []):
+            memory = _get_memory_with_deprecated_mb(preset, "memory")
+            if memory is None:
+                raise ValueError("memory is not set for resource preset")
```

Comment:
Is not it already checked in the validator?

Reply:
No, this is for cluster config get request (platform-api internal request to platform-config to fetch cluster configuration). We do not validate it.